libs/sdk-ui-dashboard/src/model/store/meta/metaSelectors.ts

```diff
@@ -184,3 +189,17 @@ export const selectDashboardShareStatus = createSelector(selectDashboardDescript
 export const selectDashboardLockStatus = createSelector(selectDashboardDescriptor, (state) => {
     return state.isLocked;
 });
+
+/**
+ * Selects complete dashboard share info.
+ *
+ * @alpha
+ */
+export const selectDashboardShareInfo = createSelector<
+    DashboardState,
```

Comment:
Detail: do we need the explicit types?

Reply:
return type needs to be defined explicitly, as I am picking particular values from whole DashboardDescriptor